docs/api/puppeteer.launch.md

```diff
@@ -7,6 +7,6 @@ sidebar_label: launch
 **Signature:**
 
 ```typescript
-launch: (options?: import('./types.js').PuppeteerLaunchOptions) =>
-  Promise<import('./types.js').Browser>;
+launch: (options?: import('./node/PuppeteerNode.js').PuppeteerLaunchOptions) =>
+  Promise<import('puppeteer-core/internal/api/Browser.js').Browser>;
```

Comment:
should not we import from the public types here and above?

Reply:
This is a problem with API extractor which existed before. We can forward fix this.